test/unit/scripts/v19-public-api-boundary.test.ts

```
@@ -1,119 +1,214 @@
-    || name === 'PatchSession'
-    || name.startsWith('createNode')
-    || name.startsWith('createEdge')
-    || name.startsWith('createProp')
-    || name === 'createInlineValue'
-    || name === 'createBlobValue'
-    || name === 'decodeEdgePropKey'
-    || name === 'encodeEdgePropKey'
-    || name === 'isEdgePropKey'
-    || GRAPH_SUBSTRATE_NOUNS.has(name);
+function sorted(values: Iterable<string>): string[] {
+  return Array.from(values).sort();
```

ESLint

[error] Use spread syntax [...iterable] instead of Array.from(iterable) when no mapper function is provided

(e18e/prefer-spread-syntax)